packages/sitecore-jss-cli/src/create/create.source.github.ts

```diff
@@ -2,7 +2,7 @@ import chalk from 'chalk';
 import fs from 'fs';
```

Comment:
@illiakovalenko Should we add unit tests for this one? Or are we assuming CLI will be covered with integration tests (separate task)?

Reply:
@ambrauer I didn't plan to cover everything, because for `sitecore-jss-cli` `sitecore-jss-dev-tools` we almost don't have unit tests. We should implement them step by stem, as I did something for `package-deploy.ts`. And yes - we are waiting for integration tests from QA side.